index.js

```diff
@@ -96,11 +103,9 @@ function init() {
       this.log.apply(this, args);
     },
 
-    err: function error() {
-      let args = Array.prototype.slice.call(arguments);
-      args.unshift('err');
-      this.log.apply(this, args);
-    },
+    err: error(),
```

Comment:
Don't call the method here. Just supply it or else it won't work as can see in the test log